### ISSUE TITLE
ICD 9 Italy

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#623]
https://forums.ohdsi.org/t/italian-icd9/16419

Ran into an interesting case today, in EHDEN we have an Italian DP who uses ICD9. And was surprised that these codes weren’t getting picked up when using ICD9CM:

'043.11', -- COVID-19 conclamata, virus identificato
‘043.21', -- COVID-19 paucisintomatica, virus identificato
'043.31', -- COVID-19 asintomatica, virus identificato
'480.41', -- Polmonite in COVID-19, virus identificato
'518.91', -- Sindrome da distress respiratorio (ARDS) in COVID-19, virus identificato
'519.71' -- Altra infezione delle vie respiratorie in COVID-19, virus identificaton`

From Christian:
Bottom line: We would have to bring it in. Shouldn’t be a big deal, really, and the mappings from the few new codes could be added manually. Just drop a Github issue.

**Comment by mik-ohdsi** [comment]
@ericaVoss  - @cgreich 
as stated / explained in the forum post as well: Strictly speaking we should not add Italian modifications to the existing US ICD9CM.
I have reached out to Matteo Gabetta to get his opinion and guidance.
Cheers
 Mik

**Comment by mik-ohdsi** [comment]
@ericaVoss  &  @cgreich 
can I close this issue if there is nothing substantial coming in the forum anymore?
